### PR TITLE
Stop using xfun::isFALSE()

### PR DESCRIPTION
**Pull request by yihui**
## Summary

This function will be deprecated in the future: https://github.com/yihui/xfun/pull/67

I pushed three commits. The first one does what the PR title says. I'm not sure if the second one makes sense to you. If not, I (or you) can revert the second one. Thanks!

## Checklist

- [x] I agree to follow the [Code of Conduct][conduct]
- [x] I have read the [contributing guidelines][contributing]
- [x] I ran the script [scripts/contribute.R][contribute.R]

[conduct]: https://github.com/workflowr/workflowr/blob/main/CODE_OF_CONDUCT.md
[contributing]: https://github.com/workflowr/workflowr/blob/main/CONTRIBUTING.md
[contribute.R]: https://github.com/workflowr/workflowr/blob/main/scripts/contribute.R

### CODE REVIEW

File: R/wflow_html.R [hunk 1]
```diff
@@ -249,15 +249,6 @@ wflow_hook_plot_md <- function(x, options) {
...
 
-  # Check for outdated reticulate
-  if (identical(options$engine, "python") &&
-      !xfun::isFALSE(options$python.reticulate) &&
-      requireNamespace("reticulate", quietly = TRUE) &&
-      utils::packageVersion("reticulate") < "1.14.9000") {
-    warnings_to_add <- c(warnings_to_add,
-                         "<strong>Warning!</strong> Please update the R package <a href=\"https://cran.r-project.org/package=reticulate\">reticulate</a> to use Python plots with workflowr")
-  }
-
```
Comment (jdblischak) [inline R/wflow_html.R:260]: I agree this code is no longer needed. It was temporary while I waited for the release of reticulate 1.15. Once it was released, I started requiring reticulate >= 1.15, at which point I should have also removed this code logic.

https://github.com/workflowr/workflowr/blob/66e6a4c0e97af3765bbe1e5d16511483a7d76ff6/DESCRIPTION#L67

**Review by jdblischak — APPROVED** [review]
Thanks @yihui!

**Comment by jdblischak** [comment]
Thanks @yihui! A few follow-up items:

1. Please let me know if you'd like to be officially added as a package contributor. You've made [multiple maintenance commits](https://github.com/workflowr/workflowr/commits?author=yihui), and crucially also answered some important design questions when I first started developing workflowr
2. I've long been meaning to use xfun more in workflowr. I like your lightweight functions for working with paths (eg `xfun::same_path()`) and calling R externally (eg `xfun::Rscript_call()`). What is the roadmap for these functions? Do you plan to support them long term?

**Comment by yihui** [comment]
Thanks @jdblischak!

1. My commits are quite trivial. It's up to you. I'll definitely feel honored to be listed as a `ctb` :)
2. Yes, most functions will be supported in the long term. `isFALSE()` is an unfortunate exception, in the sense that not long after I wrote this function, R core added `isFALSE()` to base R. Had I waited for a couple of months, I wouldn't need to add it to **xfun**. I don't like my function to clash with base R (and worse still, they have different meanings). I've been waiting for R 3.5.0 to be "old enough" so that I can just use `base::isFALSE()`. Anyway, I'm glad that you like the lightweight functions in this package, which is exactly the point of this package (zero hard dependency).

**Comment by jdblischak** [comment]
I added you as a contributor. Thanks!

> I'm glad that you like the lightweight functions in this package, which is exactly the point of this package (zero hard dependency)

I've been bitten by breaking changes in dependencies enough times to know they are a double-edged sword. Since workflowr has to depend on knitr/rmarkdown, the more functionality I can replace with xfun, the fewer potential sources of future problems

**Pull request merged**